app/src/main/res/values/strings.xml

```diff
@@ -410,8 +410,8 @@
     <string name="confirm_login_cancel_message">Are you sure you want to cancel the login process?</string>
     <string name="confirm_login_cancel_positive_button">Cancel login</string>
     <plurals name="viewers">
-        <item quantity="one">Live with %d viewer</item>
-        <item quantity="other">Live with %d viewers</item>
+        <item quantity="one">Live with %d viewer for %s</item>
```

Comment:
I think we unfortunately have to introduce a completly new string resource in order to not break the existing translations that don't have the additional argument

Reply:
Would a new string resource replace the current one?